packages/luau/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://luau-lang.org
+TERMUX_PKG_DESCRIPTION="A fast, small, safe, gradually typed embeddable scripting language derived from Lua"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.629
+TERMUX_PKG_SRCURL=https://github.com/luau-lang/luau/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=18b04e673a4e450872b57e1881fc54615f8c18addd8336b3e0b20df43562d50d
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_NO_STATICSPLIT=true
+
```

Comment:
```suggestion
TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
-DLUAU_BUILD_TESTS=OFF
-DLUAU_EXTERN_C=ON
"

```

Reply:
Please patch CLI/Repl.cpp:

```patch
--- a/CLI/Repl.cpp
+++ b/CLI/Repl.cpp
@@ -783,7 +783,7 @@ int replMain(int argc, char** argv)
     {
 #if __linux__
         char path[128];
-        snprintf(path, sizeof(path), "/tmp/perf-%d.map", getpid());
+        snprintf(path, sizeof(path), "@TERMUX_PREFIX@/tmp/perf-%d.map", getpid());

         // note, there's no need to close the log explicitly as it will be closed when the process exits
         FILE* codegenPerfLog = fopen(path, "w");
```